scripts/pool_liquidity.js

```diff
@@ -12,17 +23,38 @@ module.exports = async (deployer, network) => {
   else 
     migrations = await Migrations.deployed()
   
+  treasury = await Treasury.at(await migrations.contracts(web3.utils.fromAscii('Treasury')))
+  vat = await Vat.at(await migrations.contracts(web3.utils.fromAscii('Vat')))
+
+  console.log('Treasury')
+  const savings = await treasury.savings()
+  const debt = await treasury.debt()
+  console.log(`Dai Reserves:   ${savings.div(WAD)}`)
+  console.log(`Dai Debt: ${debt.div(WAD)}`)
+
+  const eth = (await vat.urns(web3.utils.fromAscii('ETH-A'), treasury.address)).ink
+  const spot = (await vat.ilks(web3.utils.fromAscii('ETH-A'))).spot
+  const collateral = mulRay(eth, spot)
+  console.log(`Eth Collateral: ${collateral.div(WAD)} (${eth.div(WAD)} Eth)`)
+  console.log()
+
+  let tvl = savings.add(collateral).sub(debt)
   const contracts = await migrations.length()
   for (let i = 0; i < contracts; i++) {
     const contractName = web3.utils.toAscii(await migrations.names(i))
 
     if (contractName.includes('fyDaiLP')) {
       const pool = await Pool.at(await migrations.contracts(web3.utils.fromAscii(contractName)))
       console.log(contractName)
-      console.log(`Dai Reserves:   ${await pool.getDaiReserves()}`)
-      console.log(`FYDai Reserves: ${await pool.getFYDaiReserves()}`)
+      const dai = await pool.getDaiReserves()
+      const fyDai = await pool.getFYDaiReserves()
+      console.log(`Dai Reserves:   ${dai.div(WAD)}`)
+      console.log(`FYDai Reserves: ${fyDai.div(WAD)}`)
       console.log()
+      tvl = tvl.add(dai).add(fyDai)
```

Comment:
You can't add fyDai to the TVL. 1.) It's not external value, and 2.) Most of the fyDai are virtual!

Reply:
Fixed now, thanks!